video/out/opengl/header_fixes.h

```diff
@@ -88,6 +88,15 @@
 #define GL_UNSIGNED_SHORT_8_8_REV_APPLE 0x85BB
 #endif
 
+// GL_ARB_timer_query and EXT_disjoint_timer_query
+#ifndef GL_TIME_ELAPSED
+#ifdef GL_TIME_ELAPSED_EXT
+#define GL_TIME_ELAPSED GL_TIME_ELAPSED_EXT
+#else
+#define GL_TIME_ELAPSED 0x88BF
+#endif
+#endif
+
```

Comment:
OK. but can you simplify it and always define it to 0x88BF? The constants are standardized, so it doesn't harm, and simpler is preferable.

Also, in general it's better to reference a commit hash instead of a PR number, but that doesn't need to be changed.